src/app/ReadHandler.h

```diff
@@ -144,7 +144,7 @@ class ReadHandler : public Messaging::ExchangeDelegate
     void ClearDirty() { mDirty = false; }
     bool IsDirty() { return mDirty; }
     NodeId GetInitiatorNodeId() const { return mInitiatorNodeId; }
-    FabricIndex GetFabricIndex() const { return mFabricIndex; }
+    FabricIndex GetAccessingFabricIndex() const { return mFabricIndex; }
```

Comment:
Suggest adding to WriteHandler and CommandHandler as well

Reply:
Actually, can we not add to the others, until I've got my in-flight changes in? (This is changing to a full SubjectDescriptor, and it's easier to add in final form, rather than add then change.)

Reply:
See  https://github.com/project-chip/connectedhomeip/pull/12005 and https://github.com/project-chip/connectedhomeip/pull/12040